### PR TITLE
[ADMIN 19] View teachers

### PR DESCRIPTION
**Pull request by sethamarx**
Added teachers tab and connected to back end.

### CODE REVIEW

File: src/components/TeacherLineItem/TeacherLineItem.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      <div className={styles["line-item-name-icon"]} />
+      <div className={styles["line-item-name"]}>
+        {`${teacher.firstName} ${teacher.lastName}`}
+      </div>
+    </div>
+    <div className={styles["line-item-container-right"]} />
+  </div>
+);
+
+export default TeacherLineItem;
```
Comment (jacoblurie29) [inline src/components/TeacherLineItem/TeacherLineItem.tsx:16]: Let's make some visual updates here and give each teacher a grey icon to the left of their name. Looks sort of empty without it.

File: src/pages/admin/AdminTeachersTab/AdminTeachersTab.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  );
+
+  const applicants = currentTeachers?.filter(
+    (teacher) => teacher.approvalStatus === ApprovalStatus.PENDING,
+  );
+
+  console.log(applicants);
+  return (
+    <div>
+      <h1>Admin Teachers</h1>
```
Comment (jacoblurie29) [inline src/pages/admin/AdminTeachersTab/AdminTeachersTab.tsx:53]: Change this to match the title in the Classes tab

**Review by jacoblurie29 — CHANGES_REQUESTED** [review]
Looks great! Some minor changes

File: src/pages/admin/AdminTeachersTab/AdminTeachersTab.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      <Accordion title="Teachers">
+        {teachers?.map((teacher, index) => (
+          <TeacherLineItem key={index} teacher={teacher} />
+        ))}
+      </Accordion>
+      <Accordion title="Teacher Applicants">
```
Comment (jacoblurie29) [inline src/pages/admin/AdminTeachersTab/AdminTeachersTab.tsx:59]: Give both of the accordions `noDataTitle` values

**Review by jacoblurie29 — CHANGES_REQUESTED** [review]

File: src/components/TeacherLineItem/TeacherLineItem.tsx (hunk 1 above)
Comment (sethamarx) [reply src/components/TeacherLineItem/TeacherLineItem.tsx:16]: I was thinking when we add editing here thats where it would go.

Comment (jacoblurie29) [reply src/components/TeacherLineItem/TeacherLineItem.tsx:16]: Good idea, lets leave it then

**Review by jacoblurie29 — APPROVED** [review]
LGTM!

**Pull request merged**